src/sx-design/src/SkipLink.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import * as React from 'react';
+import sx from '@adeira/sx';
+
+type Props = {|
+  +text: FbtWithoutString,
+|};
+
+// https://web.dev/headings-and-landmarks/#bypass-repetitive-content-with-skip-links
```

Comment:
Seems to be some issues with this skip-link approach on mobile devices

https://axesslab.com/skip-links/

Reply:
Do you want to send a PR with some fix? It's not clear to me what should it look like from the article.

Reply:
Looks like [this](https://github.com/selfthinker/dokuwiki_template_writr/blob/master/js/skip-link-focus-fix.js) is the best solution. 

I can try to have a look at it later, should be ok merging this now, I just wanted to point out that it might not work on all devices 🙈